pynicotine/gtkgui/ui/settings/uploads.ui

```diff
@@ -292,6 +294,7 @@
               <object class="GtkSpinButton" id="MaxUserQueue">
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
+                <property name="tooltip_text">Megabytes.</property>
```

Comment:
Technically this should be mebibytes according to the current source code, but perhaps this would cause confusion for a new user? Both are close enough to barely make a real world impact.